Solutions/Threat Intelligence/Analytic Rules/DomainEntity_CommonSecurityLog.yaml

```diff
@@ -77,15 +79,15 @@ query: |
 entityMappings:
```

Comment:
Can we remove the extend statement from the queries as they are no longer required.